submit_and_compare/static/js/submit_and_compare_view.js

```diff
@@ -23,10 +25,15 @@ function SubmitAndCompareXBlockInitView(runtime, element) {
     var hint;
     var hints;
     var hint_counter = 0;
-	
-    var cached_answer_id = question_prompt.parent().parent().attr('data-usage-id') + '_cached_answer';
+
+    var xblock_id = question_prompt.parent().parent().attr('data-usage-id');
```

Comment:
hmm, `.parent().parent()` seems pretty brittle, is there no other way to grab it?